app/src/main/java/fr/neamar/kiss/forwarder/Favorites.java

```diff
@@ -322,7 +322,10 @@ public boolean onTouch(View view, MotionEvent motionEvent) {
         int intStartX = motionEvent.getHistorySize() > 0 ? Math.round(motionEvent.getHistoricalX(0)) : intCurrentX;
         boolean hasMoved = (Math.abs(intCurrentX - intStartX) > MOVE_SENSITIVITY) || (Math.abs(intCurrentY - intStartY) > MOVE_SENSITIVITY);
 
-        if (hasMoved) {
+        // Get the number of favs, if its less than 2 don't do drag and drops.
+        int favSize = KissApplication.getApplication(mainActivity).getDataHandler().getFavorites(3).size();
```

Comment:
Do we need to get this value on the onTouch()?
That will do an SQL query for every tick when dragging.

Can't we instead disable drag completely when there is only one item?

(just asking, I can merge as-is too)

Reply:
Good point, Ill rework it a little